DMBS/gcc.mk

```diff
@@ -26,7 +26,7 @@ F_CPU              ?=
 C_STANDARD         ?= gnu99
 CPP_STANDARD       ?= gnu++98
 C_FLAGS            ?=
-CPP_FLAGS          ?=
+CPP_FLAGS          ?= -fno-exceptions
```

Comment:
I noticed that this location is actually wrong. It need to move to BASE_CPP_FLAGS instead.